platform/ios/MGLMapView.mm

```diff
@@ -383,6 +383,40 @@ - (void)commonInit
     }];
 }
 
+- (NS_ARRAY_OF(NSDictionary *) *)featuresAt:(CGPoint)point
+{
+    // flip y for core
+    point.y = self.bounds.size.height - point.y;
+
+    mbgl::FeatureResults results = _mbglMap->featuresAt(mbgl::PrecisionPoint(point.x, point.y), 50);
```

Comment:
The radius should be configurable. A 50-pixel radius (is that 25 or 50 points?) may not make sense for all use cases.